### ISSUE TITLE
Paste images (from clipboard)

### ISSUE THREAD

**Issue by vincentcox** [ehrishirajsharma/SwiftnessX#36]
**Describe the solution you'd like**
Is it possible to allow pasting images into SwiftnessX? (For example ctrl+V to paste)

Currently, images are only insert-able via the image icon, but it would save time to just paste images from the clipboard.

**Comment by ehrishirajsharma** [comment]
I've felt the need for this too... Let me see what would be the best way to implement it as QuillJS doesn't support it, neither drag & drop utility. 

I found this module: https://github.com/kensnyder/quill-image-drop-module/ to solve this, let me confirm if it is working as expected.

If everything goes well, I will add this to the upcoming release.

**Comment by vincentcox** [comment]
Unfortunately, I am not experienced with ElectronJS apps. I can however help you with the testing part (on full builds). 

Currently I am using SwiftnessX for taking notes during my OSCP labs, and the only thing that keeps me from thinking to switch back to the oldfashioned Cherrytree is pasting images. I know it takes a few clicks to add the media via the interface, but for the exam's you are limited in time and you have to cut wherever possible.

**Comment by ehrishirajsharma** [comment]
> ....and the only thing that keeps me from thinking to switch back to the oldfashioned Cherrytree is pasting images...

Yes, I understand this frustation and editor was actually meant to speed up the process, not to make it complex..

There are a lot of functionalities missing with the open-source editor we initially chose: QuillJS; however it is highly customizable and making it to match with any other rich editors, would take some time.

**RE Clipboard Image Support:** It is not that really hard to implement and can be implement but it will only be released along with the v0.3 of SwiftnessX (which is aimed for 1-2months)

If this missing module is really frustating and you can't wait till next release, I can invite you to the private-repo I maintain and commit this update in next 1-2 weeks. Let me know if you're interested.

**Comment by vincentcox** [comment]
Yeah sure, would be awesome to be invited. We can DM' via https://twitter.com/vincentcox_be.

**Comment by GlitchWitchIO** [comment]
Hey there, I recently switched to swiftness from cherrytree and am also really looking forward to clipboard image support. 

I'm starting the PWK course in preparation for the OSCP in two weeks and would love to get an invite to the private repo so that I can continue using swiftness during the course. My twitter handle is the same on twitter and DM's are open. Thanks!

**Comment by vincentcox** [comment]
Is it possible to release a version where the clipboard pasting is implemented? I know you have other things on the roadmap, but the clipboard pasting would be so handy!
No need to finish the rest, but it would be awesome if you find some time to publish a version with that feature. Thanks 💪
